### PR TITLE
Fixed `Add` expression to use GT behavior

### PR DESCRIPTION
**Pull request by metasim**
Created new BiasedAdd for Aggregate use.

Addresses: https://github.com/locationtech/rasterframes/issues/229

Related: #225 #230

### CODE REVIEW

File: core/src/main/scala/org/locationtech/rasterframes/expressions/localops/BiasedAdd.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  override protected def op(left: Tile, right: Double): Tile = DataBiasedOp.BiasedAdd(left, right)
+  override protected def op(left: Tile, right: Int): Tile = DataBiasedOp.BiasedAdd(left, right)
+
+  override def eval(input: InternalRow): Any = {
+    if(input == null) null
+    else {
+      val l = left.eval(input)
+      val r = right.eval(input)
+      if (l == null && r == null) null
+      else if (l == null) r
```
Comment (vpipkt) [inline core/src/main/scala/org/locationtech/rasterframes/expressions/localops/BiasedAdd.scala]: Hmm i am pretty lost with what this means all taken together.

Comment (metasim) [reply core/src/main/scala/org/locationtech/rasterframes/expressions/localops/BiasedAdd.scala]: @vpipkt It's for the case when one or the other are completely `null` Tiles.... no tiles.

Comment (vpipkt) [reply core/src/main/scala/org/locationtech/rasterframes/expressions/localops/BiasedAdd.scala]: not going to fall on my sword about it but a unit test around the intended behavior would be nice to see.

**Comment by vpipkt** [comment]
@metasim See my latest commit. I'm pretty happy about the Travis CI passing (Whoop) 😅

FWIW Circle passes; `it:test` passes locally for me

File: core/src/main/scala/org/locationtech/rasterframes/expressions/localops/BiasedAdd.scala (hunk 1 above)
Comment (metasim) [reply core/src/main/scala/org/locationtech/rasterframes/expressions/localops/BiasedAdd.scala]: https://github.com/locationtech/rasterframes/issues/237

**Pull request merged**